nservicebus/messaging/delayed-delivery.md

```diff
@@ -5,10 +5,21 @@ tags:
 - Defer
 ---
 
-When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered. Before version 6 this was achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
+When a message is sent it does not need to be dispatched immediately. it can be specified a later time for the message to be delivered. In Version 5 and below this can be achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In Version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
 
 NOTE: Only send operations can be deferred. You cannot defer a publish or a reply. 
 
+## Difference between versions
+### Defer Handling
+In Versions 3-5 by calling `bus.Defer(...)` the handling of the message is deferred by certain amount of time. After that time the message is passed to be handled by the same endpoint, it means that endpoint need to have handler for deferred message to be able to handle it. Using this mechanism delayed delivery can be achieved by calling `bus.Defer(...)` and a having handler in which the actual `bus.Send(...)` will be called after elapsed amount of time. 
```

Comment:
Confused - why is it important to point out that you need to make sure you have a handler that can handle that message? Is it different from other messages (e.g. it will compile even if there's no handler that handles it)? If not then I'd skip that info.